CONTRIBUTING.md

```diff
@@ -109,6 +109,18 @@ following when creating your pull request:
 
 A reviewer will be assigned automatically when your PR is created.
 
+We use [bors](https://github.com/bors-ng/bors-ng) merge bot to ensure that the `master` branch never breaks.
+Once a PR is approved, you can comment on it with the following to add your PR to the merge queue:
+
+```
+bors merge
```

Comment:
so for now the only thing that changes is instead of pressing the big green button, we comment `bors merge`? Just to double check my understanding is correct.

Reply:
Yep, that's correct.